pkg/nmi/server/redirector_windows.go

```diff
@@ -0,0 +1,108 @@
+// +build windows
+
+package server
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+)
+
+type WindowsRedirector struct {
+	Server *Server
+}
+
+func makeRedirectorInt(server *Server) RedirectorInt {
+	return &WindowsRedirector{Server: server}
+}
+
+var podMap = make(map[types.UID]string)
+
+func (s *WindowsRedirector) RedirectMetadataEndpoint() {
```

Comment:
>RedirectMetadataEndpoint [](start = 28, length = 24)

Nit: please comments to public methods

Reply:
Updated

Reply:
Do you have Go linter in your VS code? Comments on public methods should follow certain format.

---
In reply to: [382734509](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r382734509) [](ancestors = 382734509)

Reply:
yes. installed go outlinter. Will rewrite the comments

---
In reply to: [383483933](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r383483933) [](ancestors = 383483933,382734509)